crates/trigger/src/cli.rs

```diff
@@ -64,12 +64,12 @@ where
         )]
     pub follow_components: Vec<String>,
 
-    /// Print all component output to stdout/stderr
+    /// Silence all component output to stdout/stderr
     #[clap(
-        long = "follow-all",
+        long = "silent",
         conflicts_with = FOLLOW_LOG_OPT,
         )]
-    pub follow_all_components: bool,
+    pub silence_component_logs: bool,
```

Comment:
I wonder if we are at the stage where `--follow-all` is muscle memory enough for people that we should continue to allow the flag as a "no op and deprecation warning."  Not a blocker, just raising it as a thought.

Reply:
Good thought. Wonder if this will clutter peoples logs.

Reply:
Added a deprecation notice. Please lmk what you think.

Reply:
Looks good - thanks!

Reply:
Should we remove the deprecation notice before cutting 1.0?

Reply:
Yes, that would be a 4-6 week timeframe, long enough for people who use it routinely enough to have muscle memory to adjust.